web/modules/custom/portland/portland.module

```diff
@@ -13,10 +13,9 @@ use Drupal\Core\Cache\Cache;
 // This file concerns itself with setting up tokens with hooks
 include 'portland.tokens.inc';
 
-
-// The plural is used in the list page URL. E.g. "/bds/services"
-// The group list view's path must match the plural form shown here.
-// If the group list view doesn't exist, visiting the alias will get a 404.
+/**
```

Comment:
I like the move to block comments, but Drupal coding standards still suggest a line length of 80 characters—which makes PR review a lot easier as well. https://www.drupal.org/node/1354

Reply:
Trimmed block comments